context.md

```diff
@@ -348,16 +429,21 @@ sayHello.call(cat, "peachpuff");
 
 [More information](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply)
 
+--------
```

Comment:
the bind, call, and apply examples here are super contrived. maybe a better example would be to demonstrate how `forEach` works, e.g.:

```js
function forEach(collection, callback){
  for(var i = 0; i < collection.length; i++){
    callback.apply(collection[i],[collection[i],i])
  }
}

forEach(["a","b","c"], function(letter, index){
  console.log(this, letter, index)
})
```